lms/envs/common.py

```diff
@@ -450,9 +450,14 @@
         'read': 'Read scope',
         'write': 'Write scope',
         'email': 'Email scope',
-        'profile': 'Profile scope',
-    }
+        # conform profile scope message that is presented to end-user
+        # to lms/templates/provider/authorize.html
+        'profile': 'Read your user profile',
+    },
 }
+# This is required for the migrations in oauth_dispatch.models
+# otherwise it fails saying this attribute is not present in Settings
+OAUTH2_PROVIDER_APPLICATION_MODEL = 'oauth2_provider.Application'
```

Comment:
Sorry for not being clear.  The lines linked in the `test.py` file should be removed with this PR - since the addition of this line in `common.py` makes the `test.py` code unnecessary.